### ISSUE TITLE
Deleting an item that hasn't been bid on somtimes throws error

### ISSUE THREAD

**Issue by cbierman** [Pizza-and-Bier/lunafest-dyb-frontend#59]
This is because the "key" property isn't set on creation. We should probably just set it, and ensure it exists before we delete it.

**Comment by apizzimenti** [comment]
Do you want me to work on this one?

**Comment by cbierman** [comment]
yeah sure if you want to. For creation it'll be in the "new-item" component, and for the delete one it'll be somewhere in the "edit-item" component

hmu on Slack if you have any questions

**Comment by cbierman** [comment]
Ooof I lied on creation since we won't have it initially, we'll just need to set the `key` we get back from the Firebase `push()`
